skiko/src/awtMain/kotlin/org/jetbrains/skiko/SystemTheme.awt.kt

```diff
@@ -1,10 +1,13 @@
 package org.jetbrains.skiko
 
 actual val currentSystemTheme: SystemTheme
-    get() = when (getCurrentSystemTheme()) {
-        0 -> SystemTheme.LIGHT
-        1 -> SystemTheme.DARK
-        else -> SystemTheme.UNKNOWN
+    get() {
+        Library.load()
```

Comment:
It shouldn't be called on each call

Reply:
I wouldn't say "shouldn't" (`Library.load` protects against being called multiple times), but yes, it's slightly faster not to.